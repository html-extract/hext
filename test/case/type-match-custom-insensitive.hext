<body>
  { <*:type-matches(/^CuStOmTaG$/i) @text:content /> }
</body>
