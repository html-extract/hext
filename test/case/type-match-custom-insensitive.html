<html>
  <head>
  </head>
  <body>
    <p>nope1</p>
    <div>
      <p>nope2</p>
      <P>nope3</P>
      <h1>nope4</h1>
      <h2>nope5</h2>
      <h3>nope6</h3>
      <h4>nope7</h4>
      <h5>nope8</h5>
      <h6>nope9</h6>
      <h7>nope10
        <CuStOmTaG>match1</CuStOmTaG>
        <CuStOmTag>match2</CuStOmTag>
      </h7>
    </div>
    <customtag>match3</customtag>
  </body>
</html>
