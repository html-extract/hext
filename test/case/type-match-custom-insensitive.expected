{"content":["match3","match1","match2"]}
